### DISCUSSION TITLE
The tsconfig and .prettierrc files are missing in the catalyst monorepo.

### DISCUSSION THREAD

**Discussion by prasanthpillai** [bigcommerce/catalyst#1712] [Q&A] [answered]
The required configuration files, such as` tsconfig.json `and` .prettierrc` , are missing in the Catalyst monorepo, causing errors during the build process. Could you please assist with a solution?

**Comment by chanceaclark** [comment]
Can you add some more details here, like what version of Catalyst and the client you are working with? Also, an example repo would be extremely helpful here.

On main, we do have these files, so not sure what is missing from this picture:
- https://github.com/bigcommerce/catalyst/blob/main/core/tsconfig.json
- https://github.com/bigcommerce/catalyst/blob/main/core/prettier.config.js

**Reply by prasanthpillai** [reply]
In previous versions of Catalyst, the `npm create @bigcommerce/catalyst@latest` command would, by default, generate only a standalone Next.js project. 
However, it now generates the complete monorepo structure, and the root of the monorepo does not include tsconfig or Prettier configuration files.
![image](https://github.com/user-attachments/assets/fac3b72d-13dc-447e-b9c5-428ebef5ea20)
![image](https://github.com/user-attachments/assets/d98085d7-1b89-4564-94d1-2ef1903721b0)

**Reply by chanceaclark** [reply, marked as answer]
Make sure to copy over the vscode settings file we have: `cp .vscode/settings.example.json .vscode/settings.json`. After reloading your VSCode window, you should have all the config needed for those to work.
